macOSUpgrade.sh

```diff
@@ -362,6 +371,16 @@ if [[ ${pwrStatus} == "OK" ]] && [[ ${spaceStatus} == "OK" ]]; then
         /Library/Application\ Support/JAMF/bin/jamfHelper.app/Contents/MacOS/jamfHelper -windowType utility -title "$title" -icon "$icon" -heading "$heading" -description "$description" -iconSize 100 &
         jamfHelperPID=$!
     fi
```

Comment:
If https://github.com/kc9wwh/macOSUpgrade/pull/86#issuecomment-460923496 is correct,
I think that we should not give choices to the type of user dialog.